Adding first draft for DaSCH news item (#3651)
galaxyproject/galaxy-hub: content/news/2026-02-03-DaSCH/index.md

In this blog post, we’d like to present a concrete example of collaboration between these two. So let’s dive in!

## Extract text from historical prints

You can use Galaxy to analyse data from a wide range of sources, e.g. from DaSCH. 
One of the datasets accessible at DaSCH is the “Bilderfolgen Basler Frühdrucke”. 
It consists of 20 richly illustrated incunabula, i.e. scans of prints from the earliest stages of book printing, from the 15th century. 

We picked the [Ship of Fools (Narrenschiff)](https://ark.dasch.swiss/ark:/72163/1/0803/cpQ3=JfqVZOkd7hUQ26kTg7.20110414T075758Z), a satirical allegory in German verse published in 1494 in Basel. 
**It was the most popular German book before the Reformation.**
It is a moral satire about 100 fools on a boat trip, which holds up a critical and satirical mirror to the world through an entertaining description of their vices and oddities. 
While the book’s scans are very precious, their texts are not machine-readable, so they cannot be used for further analysis without retrieving a full-text from another source. 
At this point, Galaxy comes in.

We created a workflow to make the scans from the incunabula machine-readable. 

> The scans can be accessed by visiting [https://app.dasch.swiss/](https://app.dasch.swiss/), navigating to “Incunabula”, choosing “Book”, selecting “[Das] Narrenschiff (dt.)”, and then leafing through the pages. 

Reviewer: We should really get dasch integrated as Galaxy repo :)

The current analysis started with page 19. The link to an image can be obtained by clicking on “Share” and then “Copy IIIF URL to clipboard”.

<div align="center">
  <img src="image1.png" alt="View of Ship of fools on DaSCH" width="600"/>
</div>

> We can use the links from DaSCH to upload the images to Galaxy. To process all images in the same way, we create a collection of the 10 images we want to work with. 

Reviewer: We have FAQs in the GTN: https://training.galaxyproject.org/training-material/faqs/galaxy/collections_build_list.html would a link to this help?

---
title: "Connecting Humanities Data: Analyzing DaSCH data on Galaxy"
date: "2026-02-03"

> tease: "We are happy to announce the collaboration between DaSCH, the Swiss National Data and Service Center for the Humanities, and the data analysis platform Galaxy."

Reviewer 2: ```suggestion
tease: "We're excited to announce the collaboration between DaSCH, the Swiss National Data and Service Center for the Humanities, and the data analysis platform Galaxy."
```

subsites: [global,eu,us]
main_subsite: freiburg
tags: [DaSCH, repositories, data-commons, workflow, eosc, OCR, LLMHub]
contributions:
  authorship:
    - jnussbaum
    - Sch-Da
  funding:
    - deKCD
    - datacommons
    - deNBI

---

# Connecting Humanities Data: 
## Analyzing DaSCH data on Galaxy

We are happy to announce the collaboration between [DaSCH](https://dasch.swiss/), the Swiss National Data and Service Center for the Humanities, and the data analysis platform [Galaxy](https://usegalaxy.eu/). 
While DaSCH provides loads of research data, Galaxy has lots of tools and workflows for analysing it - sounds like a match made in heaven!

> # DaSCH as a data provider

Reviewer 2: A md document can only contain 1 Heading1, at the top. The others must be Heading2 or 3. It is like a tree structure.

While this might sound like a formalistic argument, I think a clean hierarchy of titles also looks nicer when rendered.

```suggestion
## DaSCH as a data provider
```

<div align="center">
  <img src="image1.png" alt="View of Ship of fools on DaSCH" width="600"/>
</div>

We can use the links from DaSCH to upload the images to Galaxy. To process all images in the same way, we create a collection of the 10 images we want to work with. 

<div align="center">
  <img src="image4.png" alt="Ship of fools as a collection on Galaxy" width="600"/>
</div>

You can follow the steps described below in this [published history](https://usegalaxy.eu/u/schnda/h/transcription-with-llm-hub-dasch-example).
And you can use [this workflow](https://usegalaxy.eu/u/schnda/w/transcription-narrenschiff-deutsch-dasch) if you want to reproduce the steps on Galaxy.

There are many ways to perform optical character recognition (OCR) on images - here we only present one of several approaches offered in Galaxy.
The [LLMHub](https://galaxyproject.org/news/2025-10-10-llm-hub/) allows running various open-source LLMs to extract text from images. We decided to use the NuMarkdown image model.

<div align="center">
  <img src="image3.png" alt="The LLMHub showing configurations to set up NuMarkdown" width="600"/>
</div>

> It returns the recognised text in Markdown format, so that the structure of the original text is maintained. One of NuMarkdown's strengths lies in its structured output. The reasoning steps are clearly separated from the actual answer by XML tags like \<think\> and \<answer\>.

Author: then we take the best practices, thanks
